app/controllers/katello/api/rhsm/candlepin_proxies_controller.rb

```diff
@@ -485,6 +499,8 @@ def authorize_proxy_routes
         User.consumer?
       when "rhsm_proxy_deleted_consumers_path"
         current_user.admin?
+      when "rhsm_proxy_jobs_get_path"
+        User.consumer? || current_user.admin?
```

Comment:
could you please briefly explain what this method authorizes and why some routes can be accessed only by admin? it might be unrelated but I want to understand whether the async update will be still doable under service account with minimal set of permissions